carbon-design-system/carbon-addons-iot-react
# feat(card): Card actions support to custom content

**Summary**

- Adding support to card actions to array with custom content, so we can send custom elements to be used

**Change List (commits, features, bugs, etc)**

- extraActions support to array
- updates in MDX

**Acceptance Test (how to verify the PR)**

- Check that custom content can be sent in the extraActions prop following the pattern of array with objects

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Check that none of the changes impacted the card header



**Things to look for during review**
